app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
   before_action :require_signin_permission!
 
-  protect_from_forgery with: :exception
+  rescue_from ActionController::ParameterMissing do |e|
+    render json: { error: { message: e.message } }, status: 400
+  end
+
+  rescue_from ActiveRecord::RecordNotFound do |e|
+    render json: { error: { message: e.message } }, status: 404
+  end
```

Comment:
If I don't put this in, the `ActiveRecord::RecordNotFound` error gets passed up all the way to the test, so it seems like we do?